webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] [OPTIONAL] Let's avoid disabling eslint in our code base as it helps and guides us to write standard and acceptable code.